recipe/meta.yaml

```diff
@@ -21,23 +22,31 @@ requirements:
     - wheel
   run:
     - python
-    - matplotlib-base >=3.2
-    - numpy >=1.21.0
+    - matplotlib-base >=3.5
+    - numpy >=1.22.0,<2.0
     - scipy >=1.8.0
-    - pandas >=1.3.0
+    - pandas >=1.4.0
     - xarray >=0.21.0
     - h5netcdf >=1.0.2
     - packaging
     - typing_extensions >=4.1.0
     - xarray-einstats >=0.3
 
 test:
+  source_files:
+    - arviz
   imports:
     - arviz
   requires:
     - pip
+    - pytest
+    - pytest-cov
+    - cloudpickle
+    - bokeh >=1.4.0
```

Comment:
`bokeh>=1.4.0,<3.0`, see https://github.com/arviz-devs/arviz/blob/v0.17.1/requirements-optional.txt#L3-L3C18

Reply:
I get a dependency error on 3.10 with that pinning, but I'm honestly not sure why, since manually creating a conda environment with all of the dependencies works just fine.

```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform osx-arm64:
 {MatchSpec("bokeh==2.4.3=py310hca03da5_0"), MatchSpec("python[version='>=3.10,<3.11.0a0']")}
```

Reply:
this dependency is only for testing purposes. I would say that the pinning can be relaxed.